src/init.cpp

```diff
@@ -441,6 +441,7 @@ std::string HelpMessage(HelpMessageMode mode)
         strUsage += HelpMessageOpt("-checkmempool=<n>", strprintf("Run checks every <n> transactions (default: %u)", defaultChainParams->DefaultConsistencyChecks()));
         strUsage += HelpMessageOpt("-checkpoints", strprintf("Disable expensive verification for known chain history (default: %u)", DEFAULT_CHECKPOINTS_ENABLED));
         strUsage += HelpMessageOpt("-disablesafemode", strprintf("Disable safemode, override a real safe mode event (default: %u)", DEFAULT_DISABLE_SAFEMODE));
+        strUsage += HelpMessageOpt("-enablerpcmethod", "Enables a deprecated RPC method");
```

Comment:
I don't think validating arguments is the norm. If I'm wrong, please point out where in the code this happens.

I'd definitely concept ACK a PR for better validating of command line arguments in general (for example, warning if a user tries setting an argument that doesn't exist, to catch mis-spellings).

This particular argument `enablerpcmethod` seems to be one where validation is least useful. Users will find out instantly if they didn't enable the RPC method. Outdated `enablerpcmethod` arguments are a no-op so are harmless.